html/pfappserver/root/src/components/new/BaseFormGroupToggle.vue

```diff
@@ -54,11 +51,11 @@ import { useInputValidator, useInputValidatorProps } from '@/composables/useInpu
 import { useInputValue, useInputValueProps } from '@/composables/useInputValue'
 import { useInputValueToggle, useInputValueToggleProps } from '@/composables/useInputValueToggle'
 import BaseFormGroup from './BaseFormGroup'
-import BaseInputRange from './BaseInputRange'
+import BaseInputToggle from '@/components/new/BaseInputToggle.vue';
```

Comment:
Remove trailing semi-colon

Reply:
Hmmm... Have we concidered adding an eslint rule for this? https://eslint.org/docs/latest/rules/semi